output/example3/graphs.json

```diff
@@ -308,40 +280,6 @@
     "exj_theo": [
         1.0
     ],
-    "ey_hat": [
-        [
-            1.0,
-            NaN,
```

Comment:
It's difficult to check if the NaN are right because I can't easily see the old values..

Reply:
These are the old values. Do you mean the new values? Those can be seen in the `estimate.json`.

Reply:
Normally, we have a line to line comparison (+/-); it is just difficult to read the changes from here.